ingest/ledger_transaction.go

```
@@ -83,6 +83,16 @@ func (t *LedgerTransaction) GetChanges() ([]Change, error) {
 	return changes, nil
 }
 
+// GetOperation returns an operation by index.
+// TODO: Clean this up
+func (t *LedgerTransaction) GetOperation(index uint32) (xdr.Operation, bool, error) {
+	ops := t.Envelope.Operations()
+	if len(ops) == 0 || int(index) >= len(ops) {
+		return xdr.Operation{}, false, nil
+	}
```

Shouldn't we let it panic?